src/Integrations/Hubspot/HubspotClient.php

```diff
@@ -114,6 +119,52 @@ public function getItem(string $itemId): array
 		return $output[$itemId] ?? [];
 	}
 
+	/**
+	 * Return contact properties with cache option for faster loading.
+	 *
+	 * @return array<string, mixed>
+	 */
+	public function getContactProperties(): array
+	{
+		$output = get_transient(self::CACHE_HUBSPOT_CONTACT_PROPERTIES_TRANSIENT_NAME) ?: []; // phpcs:ignore WordPress.PHP.DisallowShortTernary.Found
+
+		if (empty($output)) {
+			$items = $this->getHubspotContactProperties();
+
+			$output = [];
+
+			$allowedTypes = array_flip([
+				'text',
+				'textarea',
+			]);
```

Comment:
This is a manual array, why not just writing

```suggestion
			$allowedTypes = [
				'text' => 0,
				'textarea' => 1,
			];
```

?